labcodes/knowledge
# [mailgun-setup] Change settings to use mailgun

**Changes**
* Changed message format to send user's email
* Added mailgun settings

**How to test**
1. Insert the mailgun credentials on `.env` file.
2. Make sure you have set `SLACK_TOKEN` on `.env` file.
3. Get you Slack User ID on [users.info](https://api.slack.com/methods/users.info/test).
4. Run the server and check if there is any user created, if there is, delete it.
###### Testing using API
* Access the endpoint `/api/link/`
* Enter a payload replacing SLACK_ID with your Slack User ID:
``` json
{
  "text": "http://test.me",
  "user_id": "<SLACK_ID>"
}
```
* If your account wasn't created before, you should receive an email with login and password
###### Testing on shell
* Enter on shell using `python manage.py shell`
* Import `get_slack_user()` function
* Run `get_slack_user(SLACK_ID)` where SLACK_ID is your Slack User ID
* Check your email with login and password
